pages/osx/lp.md

```diff
@@ -0,0 +1,15 @@
+# lp
+
+> print files
+
+- print a file to a named printer (found in Printers & Scanners --> Options and Supplies)
+
+'lp -d {{device_name}} {{path/to/filename}}
+
+- print N copies to default printer (example used is 2)
+
+'lp -n 2 {{path/to/filename}}'
+
+- print only certain pages to the default printer (print pages 1, 3-5, and 16)
+
+'lp -n -P 1,3-5,16'
```

Comment:
Ahhh! Thanks for correcting me there.  I updated the commit.

On Wed, Nov 4, 2015 at 12:28 AM, Romain Prieto <notifications@github.com>
wrote:

> In pages/osx/lp.md
> <https://github.com/tldr-pages/tldr/pull/318#discussion_r43844337>:
>
> > @@ -0,0 +1,15 @@
> > +# lp
> > +
> > +> print files
> > +
> > +- print a file to a named printer (found in Printers & Scanners --> Options and Supplies)
> > +
> > +'lp -d {{device_name}} {{path/to/filename}}
> > +
> > +- print N copies to default printer (example used is 2)
> > +
> > +'lp -n 2 {{path/to/filename}}'
> > +
> > +- print only certain pages to the default printer (print pages 1, 3-5, and 16)
> > +
> > +'lp -n -P 1,3-5,16'
>
> Just to clarify I meant in the lp -n -P example
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tldr-pages/tldr/pull/318/files#r43844337>.
>



-- 
Regards,

Michael Quatrani
215.694.3468